Apsitvarkom.DataAccess/IGeocoder.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Apsitvarkom.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
mostly all these `using` directives are not used. delete the not necessary ones.